api-compose-object.go

```diff
@@ -527,6 +527,12 @@ func (c Client) ComposeObject(dst DestinationInfo, srcs []SourceInfo) error {
 	for k, v := range metaMap {
 		metaHeaders[k] = v
 	}
+
+	// Add destination encryption headers
+	for k, v := range dst.encryption.getSSEHeaders(false) {
```

Comment:
more readable to just use GetSSEHeaders().LGTM otherwise.

Reply:
We don't need to stick to requirements of a public function. Since this is an internal functionality it makes sense to use the internal function.

Reply:
ok